.circleci/windows/test.bat

```diff
@@ -0,0 +1,66 @@
+@echo on
+@rem Required vars:
+@rem    WIN_OTP_PATH
+@rem    WIN_MSYS2_ROOT
+@rem    ERTS_VERSION
+@rem    PACKAGE_TESTS_DIR
+@rem    PACKAGE_ZIPARCHIVE
+@rem    TEST_STEPS
+@rem    BUILD_PATH
+
+SETLOCAL ENABLEEXTENSIONS
+@echo %time% : Set the paths appropriately
+
+:: Appropriately set paths to MSYS and OTP avoiding duplicates
+SET "_PATHS=%WIN_MSYS2_ROOT%\mingw64\bin;%WIN_MSYS2_ROOT%\usr\bin;%WIN_MSYS2_ROOT%"
+ECHO."%PATH%" | FINDSTR /V /C:"%_PATHS%">nul && SET "PATH=%_PATHS%;%PATH%"
+SET "_PATHS=%WIN_OTP_PATH%\bin;%WIN_OTP_PATH%\erts-%ERTS_VERSION%\bin;%WIN_OTP_PATH%\erts-%ERTS_VERSION%"
+ECHO."%PATH%" | FINDSTR /V /C:"%_PATHS%">nul && SET "PATH=%_PATHS%;%PATH%"
+SET "_PATHS="
+
+:: Find absolute unix project path and remove trailing /
+for /f %%i in ('cygpath -a ..\..') do SET "PROJECT_PATH=%%i"
+SET "PROJECT_PATH=%PROJECT_PATH:~0,-1%"
+
+IF "%BUILD_PATH%"=="" SET "BUILD_PATH=%PROJECT_PATH%"
+IF "%ARTIFACTS_PATH%"=="" SET "ARTIFACTS_PATH=%BUILD_PATH%/packages"
+
+bash -lc "cd %BUILD_PATH% && make REVISION"
+
+SET /p PACKAGE_VERSION=<%~dp0..\..\REVISION
+
+:: Set required vars defaults
+
+IF "%ERTS_VERSION%"=="" SET "ERTS_VERSION=9.3"
+IF "%PACKAGE_TESTS_DIR%"=="" SET "PACKAGE_TESTS_DIR=/tmp/package_tests"
+IF "%PACKAGE_ZIPARCHIVE%"=="" SET "PACKAGE_ZIPARCHIVE=aeternity-%PACKAGE_VERSION%-windows-x86_64.zip"
+IF "%TEST_STEPS%"=="" SET "TEST_STEPS=release"
+
+@for /f "tokens=1* delims=, " %%i in ("%TEST_STEPS%") do @(
+	call :TEST_%%i
+)
+
+@echo %time% : Finished test phase
+exit /b %ERRORLEVEL%
+
+:: Subroutines
+:TEST_ct
+@echo %time% : Run test: ct
+bash -lc "cd ${BUILD_PATH} && epmd -daemon && make ct"
+exit /b 0
+
+:TEST_eunit
+@echo %time% : Run test: eunit
+bash -lc "cd ${BUILD_PATH} && epmd -daemon && make eunit"
+exit /b 0
+
+:TEST_release
+@echo %time% : Run test: release
+bash -lc ^"cd ${BUILD_PATH} ^&^& ^
+           make python-env PIP=/mingw64/bin/pip3 ^&^& ^
+           rm -rf ${PACKAGE_TESTS_DIR} ^&^& ^
```

Comment:
I did remove `epmd -daemon` as it make CircleCI wait forever after the job finishes (I reported them this, but don't expect much) I guess CircleCI finds that there is a background process that is executing and will wait for it but will fail after 10 minutes with "No output for 10 minutes" message.
Unfortunately using 'epmd -kill' didn't fix the problem.